src/Files.App/Utils/Shell/Win32API.cs

```diff
@@ -568,18 +568,17 @@ public static Task<bool> MountVhdDisk(string vhdPath)
 
 		public static Shell32.ITaskbarList4? CreateTaskbarObject()
 		{
-			var taskbar2 = new Shell32.ITaskbarList2();
 			try
 			{
+				var taskbar2 = new Shell32.ITaskbarList2();
 				taskbar2.HrInit();
+				return taskbar2 as Shell32.ITaskbarList4;
 			}
 			catch (NotImplementedException)
```

Comment:
```suggestion
			catch (Exception)
```
FileNotFoundException should also be able to be caught.